src/main/scripts/src/app/core/header/header.component.html

```diff
@@ -11,10 +11,10 @@
       <span>Search</span>
     </button>
   </div>
-  <div class="project">
+  <div class="project" *ngIf="workspaceService.isWorkspaceSet()">
     <a routeLink="/workspace">
```

Comment:
I guess based on Martin's comments about apache reverse proxy we should add href to `a` tag

```HTML
<a href="javascript: void(0)" routerLink="/workspace">...</a>
```